proposals/lip-0044.md

```diff
@@ -56,14 +56,36 @@ We define the following constants:
 
 Name                           | Type    | Value                                                | Description
 ------------------------------ | ------- | ---------------------------------------------------- | -------------------------------------------------------------------------------
-`MODULE_ID_VALIDATORS`         | uint32  | TBD                                                  | ID of the Validators module.
+`MODULE_ID_VALIDATORS`         | bytes   | TBD                                                  | ID of the Validators module.
 `STORE_PREFIX_VALIDATORS_DATA` | bytes   | 0x0000                                               | Store prefix of the validators data substore.
 `STORE_PREFIX_GENERATOR_LIST`  | bytes   | 0x4000                                               | Store prefix of the generator list substore.
 `STORE_PREFIX_BLS_KEYS`        | bytes   | 0x8000                                               | Store prefix of the registered BLS keys substore.
 `STORE_PREFIX_GENESIS_DATA`    | bytes   | 0xc000                                               | Store prefix of the genesis data substore.
-`INVALID_BLS_KEY`              | bytes   | 48 bytes all set to 0x00                             | An invalid BLS key, used as a placeholder before a valid BLS key is registered.
+`INVALID_BLS_KEY`              | bytes   | `BLS_PUBLIC_KEY_LENGTH` bytes all set to 0x00                             | An invalid BLS key, used as a placeholder before a valid BLS key is registered.
 `BLOCK_TIME`                   | integer | 10 (value for the Lisk mainchain)                    | Block time (in seconds) set in the chain configuration.
-`MAINCHAIN_ID`                 | uint32  | Value is specified in [LIP 0045][lip-0045#constants] | The chain ID of the mainchain as part of the interoperability module.
+`MAINCHAIN_ID`                 | bytes   | Value is specified in [LIP 0045][lip-0045#constants] | The chain ID of the mainchain as part of the interoperability module.
+`TYPE_ID_GENERATOR_KEY_REGISTRATION` | bytes   | 0x0001      | Type ID of the generator key registration event. |
+`TYPE_ID_BLS_KEY_REGISTRATION` | bytes   | 0x0002      | Type ID of the BLS key registration event. |
+`KEY_REG_RESULT_SUCCESS` | uint32   | 0      | Success result code of key registration events. |
+`KEY_REG_RESULT_NO_VALIDATOR` | uint32   | 1      | Failure result code of key registration events: address not registered as validator. |
+`KEY_REG_RESULT_ALREADY_VALIDATOR` | uint32   | 2      | Failure result code of key registration events: address already registered as validator. |
+`KEY_REG_RESULT_DUPLICATE_BLS_KEY` | uint32   | 3      | Failure result code of key registration events: BLS key already registered in the chain. |
+`KEY_REG_RESULT_NO_UPDATE_BLS_KEY` | uint32   | 4      | Failure result code of key registration events: BLS keys cannot be updated. |
```

Comment:
There is no event with this type any more.

Reply:
removed